documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -335,7 +335,7 @@ Then, the API invoker polls the token endpoint by making an "HTTP POST" request
     }
     ```
 - When this response is received, the API invoker must wait the seconds of the `interval` value received in the CIBA authorization endpoint and then repeat the request until a final response is received.
-- Once the user has granted consent, the API exposure platform operator will provide the access token (OperatorAccessToken) to the API invoker (Step 8). The OperatorAccessToken issued will be encrypted so no relevant information is disclosed.
+- Once the user has granted consent, the API exposure platform operator will provide the access token (OperatorAccessToken) to the API invoker (Step 8). The OperatorAccessToken issued will be encrypted so no relevant information is disclosed. If the issued access token scope is different from the one requested by the application, based on the authorization server policy or the user's instructions, the authorization server must include the "scope" response parameter to inform the application of the actual scope granted. If the user denied the authorization request an `access_denied` error may be returned. Eventually, if the user does not respond when consent is required, the operator may return an `expired_token` error.
```

Comment:
CIBA says in https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#Introduction

> This specification does not change the semantics of the OpenID Connect Authentication flow. It introduces a new endpoint to which the authentication request is posted. It introduces a new asynchronous method for authentication result notification or delivery. It does not introduce new scope values nor does it change the semantics of standard OpenID Connect parameters.

And OIDC does not speak about changing scope, but [Oauth2](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3) does:

> If the issued access token scope
   is different from the one requested by the client, the authorization
   server MUST include the "scope" response parameter to inform the
   client of the actual scope granted.

It seems the clarification about scope does repeat what OAuth2 already defines as required behaviour.
I suggest removing "If the issued access token scope is different from the one requested by the application, based on the authorization server policy or the user's instructions, the authorization server must include the "scope" response parameter to inform the application of the actual scope granted.".

Reply:
Regarding "If the user denied the authorization request an `access_denied` error may be returned."

Please see https://openid.net/specs/openid-client-initiated-backchannel-authentication-core-1_0.html#rfc.section.11 which states:

access_denied
    The end-user denied the authorization request. 

This clarification does not add anything that is not already in the standard, right?

Reply:
Regarding "Eventually, if the user does not respond when consent is required, the operator may return an `expired_token` error."

Again already stated in the standard.

Reply:
I pretty much agree, but please have a look at https://github.com/camaraproject/IdentityAndConsentManagement/pull/114#discussion_r1504022959 where I provide the reasoning behind the content of this PR.